Client.Core/Exceptions/InfluxException.cs

```diff
@@ -56,6 +57,11 @@ public HttpException(string message, int status, Exception exception = null) : b
         /// </summary>
         public int? RetryAfter { get; set; }
 
+        /// <summary>
+        ///     The response headers
+        /// </summary>
+        public IEnumerable<HeaderParameter> Headers;
```

Comment:
I do not have a strong opinion on this, but since `Headers` property in `RestResponse` class is nullable (`T?`), I think it should be here too, ie.

```suggestion
        public IEnumerable<HeaderParameter>? Headers { get; set; }
```

Otherwise good to go after CHANGELOG is updated. The amount of test coverage of the new feature is outstanding.

Reply:
Added as suggested